datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/mirrormaker/KafkaMirrorMakerConnectorTask.java

```diff
@@ -423,6 +428,8 @@ protected void postShutdownHook() {
     List<BrooklinMetricInfo> metrics = new ArrayList<>();
     metrics.addAll(AbstractKafkaBasedConnectorTask.getMetricInfos(
         generateMetricsPrefix(connectorName, CLASS_NAME) + MetricsAware.KEY_REGEX));
+    metrics.add(new BrooklinMeterInfo(generateMetricsPrefix(connectorName, CLASS_NAME) + MetricsAware.KEY_REGEX
```

Comment:
I believe this is your intent but I'm just confirming: the use of a `BrooklinMeterInfo` with a regex + `_dynamicMetricsManager.createOrUpdateMeter()` means you only want this metric to be created and emitted lazily, i.e. 
- no InGraph will appear for it until the first time it's emitted 
- Its InGraph won't show any data when it's not being emitted

Reply:
Yeah, this is indeed the intent. Thanks for checking.